### PR TITLE
fix(explorer/graphql): fixing cursors

### PR DESCRIPTION
**Pull request by DanFlannel**
adding previous cursor and fixing cursor indices 

task: https://app.asana.com/0/1206208509925075/1207157766307988

### CODE REVIEW

File: explorer/db/testutil.go [hunk 1]
```diff
@@ -48,6 +50,7 @@ func CreateTestBlocks(t *testing.T, ctx context.Context, client *ent.Client, cou
 	var msg *ent.Msg
 	var blocks []ent.Block
 	for i := 0; i < count; i++ {
+		time.Sleep(10 * time.Millisecond)
```
Comment (pavelnikolov) [inline explorer/db/testutil.go:53]: Why is this magic wait time needed? And how do you know 10ms is enough? Is there a better way to do this?

File: explorer/graphql/app/schema.graphql [hunk 2]
```diff
@@ -216,12 +216,16 @@ type XMsgEdge {
...
 
-  "Next Page Cursor"
+  "Next Cursor"
+  NextCursor: BigInt!
+
+  "Has Next Page"
   HasNextPage: Boolean!
 
+  "Previous Cursor"
+  PrevCursor: BigInt!
```
Comment (pavelnikolov) [inline explorer/graphql/app/schema.graphql:227]: ❤️

Comment (pavelnikolov) [reply explorer/graphql/app/schema.graphql:227]: Why is the cursor of type `BigInt!`? Is it supposed to be human-readable? I thought that cursors are usually of type `graphql.ID` 🤔. The official documentation [recommends base64 encoding](https://graphql.org/learn/pagination/#:~:text=As%20a%20reminder%20that%20the%20cursors%20are%20opaque%20and%20that%20their%20format%20should%20not%20be%20relied%20upon%2C%20we%20suggest%20base64%20encoding%20them.) them and making them unique:
> As a reminder that the cursors are opaque and that their format should not be relied upon, we suggest base64 encoding them.

File: explorer/db/testutil.go (hunk 1 above)
Comment (DanFlannel) [reply explorer/db/testutil.go:53]: Just to make sure the time stamp on the block entries in the test environment are different enough to sort by, I'll add some comments good catch

**Review by pavelnikolov — APPROVED** [review]

**Comment by DanFlannel** [comment]
@pavelnikolov this was a quick fix for the FE to continue, appreciate our chatting about it, looking forward to rewriting this tomorrow

**Pull request merged**